### PR TITLE
Doc mismatch fixed

### LINKED ISSUE #13323: Documentation mismatch in Preprocessing data
## Environment info

- `transformers` version: 4.10.0.dev0
- Platform: macOS-10.16-x86_64-i386-64bit
- Python version: 3.9.2
- PyTorch version (GPU?): 1.9.0 (False)
- Tensorflow version (GPU?): 2.6.0 (False)
- Flax version (CPU?/GPU?/TPU?): 0.3.4 (cpu)
- Jax version: 0.2.19
- JaxLib version: 0.1.70
- Using GPU in script?: No
- Using distributed or parallel set-up in script?: No

### Who can help

@sgugger @SaulLu 

## Information

There seems a conflict in [ Utilities for tokenizers ](https://huggingface.co/transformers/internal/tokenization_utils.html?highlight=truncation#transformers.tokenization_utils_base.PreTrainedTokenizerBase.__call__) and [Preprocessing data](https://huggingface.co/transformers/preprocessing.html?highlight=truncation#everything-you-always-wanted-to-know-about-padding-and-truncation).
In **Preprocessing data**, For **`truncation_strategy = True`**, It states "truncate to a maximum length specified by the max_length argument or the maximum length accepted by the model if no max_length is provided (max_length=None). This will only truncate the first sentence of a pair if a pair of sequences (or a batch of pairs of sequences) is provided."
whereas for the same in **Utilities for tokenizers**, it states "Truncate to a maximum length specified with the argument max_length or to the maximum acceptable input length for the model if that argument is not provided. This will truncate token by token, removing a token from the longest sequence in the pair if a pair of sequences (or a batch of pairs) is provided.".

## Expected behavior

In Preprocessing_data documentation , `truncation_strategy=True` must match with `longest_first` instead of `only_first`.

### PR DESCRIPTION
**Pull request by Apoorvgarg-creator**
# What does this PR do?

Fixes #13323

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@sgugger

### CODE REVIEW

**Review by sgugger — APPROVED** [review]
Thanks a lot!
